source/github-helpers/github-url.ts

```diff
@@ -44,8 +44,9 @@ export default class GitHubURL {
 		const filePath = ambiguousReference.slice(1).join('/');
 
 		const currentBranch = getCurrentBranch();
-		const currentBranchSections = currentBranch.split('/');
+		const currentBranchSections = currentBranch?.split('/');
 		if (
+			!currentBranch || // Current page does not have a branch; GitHubURL probably shouldn’t be used here
```

Comment:
This should fix the types
```suggestion
			!currentBranchSections || // Current page does not have a branch; GitHubURL probably shouldn’t be used here
```